package.json

```diff
@@ -53,15 +53,16 @@
     "gulp-connect": "^5.5.0",
     "gulp-css-flip": "^0.4.0",
     "gulp-css-inline-images": "^0.1.1",
-    "gulp-sass": "^4.1.0",
+    "gulp-sass": "^5.0.0",
     "gulp-util": "^3.0.7",
     "protractor": "^7.0.0",
     "streamqueue": "^1.1.1",
     "webdriver-manager": "^12.1.8"
   },
   "dependencies": {
     "dialog-polyfill": "^0.4.7",
-    "material-design-lite": "^1.2.0"
+    "material-design-lite": "^1.2.0",
+    "sass": "^1.49.7"
```

Comment:
Is this needed here? I assume this is only needed for the build process (in gulpfile only). So only `devDependencies` need to be updated?

Reply:
These got autogenerated when i ran `npm run demo`. I tried reverting, but they regenerated, so added them in the PR.

Reply:
I think `npm install sass` i adding this. You should run `npm install sass --save-dev`

Reply:
You are right. Removed this line and ran`npm install sass --save-dev` to run `npm install` successfully.